using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -705,7 +813,41 @@ If you have an [Experience Cloud ID](https://app.gitbook.com/@aep-sdks/s/docs/us
 {% endhint %}
 
 {% tabs %}
-{% tab title="iOS" %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### getTrackingIdentifier
+
+Retrieves the Analytics tracking identifier.
+
+#### Syntax
+
+```swift
+static func getTrackingIdentifier(completion: @escaping (String?, Error?) -> Void)
+```
+
+#### Example
+
+**Swift**
+
+```swift
+Analytics.getTrackingIdentifier { (trackingId, error) in
+   // Handle the error (if non-nil) or use the trackingId value. 
+}
+```
+
+**Objective-C**
+
+```objectivec
+AEPMobileAnalytics getTrackingIdentifier:^(NSString * _Nullable trackingIdentifier, NSError * _Nullable error) {
+   // Handle the error (if non-nil) or use the trackingId value.  
```

Comment:
.. or use the trackingIdentifier value

Please note that this comment talks about the trackingIdentifier parameter. Make sure all comments below follow this rule.